### ISSUE TITLE
Possible issue with project folder not in root

### ISSUE THREAD

**Issue by mokagio** [openbakery/gradle-xcodePlugin#287]
Hello, let me start by saying that I'm very new to the Gradle ecosystem, so it's very probable that this might not actually be an issue. If so, sorry about the trouble, and thanks for educating me 😄 

My theory is that the plugin is not supporting running `xcodebuild` for projects located in a subfolder rather than in the current directory.

**Example:** running `gradle xcodebuild` works in this case:

```
MyApp/
  |
  |_ MyApp.xcodeproj
```

But doesn't work in this:

```
MyProject/
  |
  |_ MyApp/
    |
    |_ MyApp.xcodeproj
```

I created [this example project](https://github.com/mokagio/GradleXcodeIssueDemo) to reproduce my issue.

**Comment by renep** [comment]
The build.gradle musst be in the same directory as the *.xcodeproj. 
I have added some time ago that the location of the project directory can be set with
```
xcodebuild {
  projectFile = 'MyApp'
}
```
But I do not use this in any of my projects, so I cannot say if this is working properly, because this is a work in progress to support multiproject builds.

**Comment by bamx23** [comment]
You could specify subfolder in settings.gradle
See https://docs.gradle.org/current/userguide/multi_project_builds.html

**Comment by phatblat** [comment]
@bamx23 there was an issue preventing builds from working in a gradle subproject, but that has been fixed by #344

**Comment by phatblat** [comment]
@mokagio I've created a fix for this in #345. You can test this out in [my fork[(https://github.com/phatblat/GradleXcodeIssueDemo) of your example project.

**Comment by mokagio** [comment]
Hey @phatblat sorry I didn’t followup on this. 😞

I am not longer working with this plugin or the company that used it. If the core contributors are happy with your fix I think this issue could be closed.

Thanks 😊
